### ISSUE TITLE
Exacerbate by  stress

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10635]
**Preferred term label:**

**Synonyms**
Aggravated by stress

**Definition (free text, please give PubMed ID)**
PMID: 37545821 The first clinical symptoms of Darier disease usually appear in adolescence or early adulthood and may be associated with pruritus. Different factors are thought to exacerbate keratosis follicularis. Some of them include exposure to ultraviolet radiation, mechanical injuries, surgery procedures, humidity, heat, stress, menstruation, pregnancy and childbirth.

**Parent term (use hpo.jax.org/app)**
Aggravated by HP:0025285

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:218 DARIER DISEASE

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
"stress" is such a general term and it is so hard to define what it means that I am not convinced this is a great HPO term. Probably a great deal of symptoms of many diseases are made worse by "stress"...
